.github/workflows/1.249-lcm.yml

```diff
@@ -0,0 +1,84 @@
+name: Build and test (1.249-lcm, scheduled)
+
+on:
+  schedule:
+    # run every Monday, this refreshes the cache
+    - cron: '13 2 * * 1'
+
+jobs:
+  python-test:
+    name: Python tests
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        test: ["", "-3"]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: '1.249-lcm'
```

Comment:
All checkout actions must use the ref `1.249-lcm` to checkout the correct branch.